src/pdf.js

```diff
@@ -52,6 +52,7 @@ import {
   version,
 } from "./display/api.js";
 import {
+  changeLightness,
```

Comment:
Now that this is not a private function anymore and we effectively use it as API, could you add a unit test to https://github.com/mozilla/pdf.js/blob/master/test/unit/display_utils_spec.js so that it's covered by tests to avoid regressions?